Bindings/bindings.json

```diff
@@ -50,7 +51,8 @@
                     "resource": "EventHub",
                     "required": true,
                     "label": "Event Hub connection",
-                    "help": "This is the connection string for your Event Hub. Make sure that it has Receive permissions."
+                    "help": "This is the connection string for your Event Hub. Make sure that it has Receive permissions.",
```

Comment:
This connection string in for service bus, ,may be the text "This is the Service Bus connections string" might be more suitable.